content/en/docs/Installation/_index.md

```diff
@@ -55,6 +55,45 @@ To install the Kyverno [Pod Security Standard policies](/policies/pod-security/)
 helm install kyverno-policies kyverno/kyverno-policies -n kyverno
 ```
 
+### Notes for ArgoCD users
+
+When deploying this chart with ArgoCD you will need to enable `Replace` in the `syncOptions`, and you probably want to ignore diff in aggregated cluster roles.
```

Comment:
The statement "and you probably want to ignore diff in aggregated cluster roles" needs more clarification. Even I'm not clear what this means and what I should do.